formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -88,7 +88,7 @@ let x = 10 [@on10] - 20;
 
 let x = 10 [@on10] - 20;
 
-let x = (-) 10 20 [@attrEntireEverything];
```

Comment:
I kind of liked it better in my first round of changes - if applying attributes to infix operations, then render the infix applications as normal-fix. Oh well, we'll revise attribute syntax anyways.